partials/mobile-nav.hbs

```diff
@@ -0,0 +1,116 @@
+            </li>
+            <li>
+                <a href="/tools-and-libraries"><span>Tools &amp; Libraries</span> <span>Curated list of useful tools</span></a>
+            </li>
+            <li>
+                <a href="/web3"><span>Web3</span> <span>Prototypes and patterns using Holochain</span></a>
+            </li>
+        </ul>
+    </li>
+    <li id="community_page_item" class="menu_item"><button>COMMUNITY</button>
+        <ul class="sub-menu">
+            <li>
+                <a href="/events"><span>Upcoming Events</span> <span>Connect with us online and IRL</span></a>
+            </li>
+            <li>
+                <a href="/get-involved"><span>Get Involved</span> <span>Discover a diverse, global ecosystem</span></a>
+            </li>
+            <li>
+                <a href="https://developer.holochain.org/" target="_blank" rel="noopener noreferrer"><span>Learn</span> <span>Tutorials and technical resources</span></a>
+            </li>
+            <li>
+                <a href="https://blog.holochain.org/" target="_blank" rel="noopener noreferrer"><span>Stay
+                        Informed</span> <span>Blog, annoucements, and newsletter</span></a>
+            </li>
+        </ul>
+    </li>
+    <li id="about_page_item" class="menu_item"><button>ABOUT</button>
+        <ul class="sub-menu">
+            <li>
+                <a href="/foundation"><span>Holochain Foundation</span> <span>Explore Holochains approach</span></a>
+            </li>
+            <li>
+                <a href="https://holo.host/" target="_blank" rel="noopener noreferrer"><span>Holo</span> <span>A P2P distributed platform for hosting</span></a>
+            </li>
+            <li>
+                <a href="https://opensource.org/licenses/CAL-1.0" target="_blank" rel="noopener noreferrer"><span>Holochain Open Source License</span> <span>Cryptographic Autonomy License (CAL)</span></a>
+            </li>
+            <li>
+                <a href="/intellectual-property"><span>Holochain Intellectual Property</span> <span>Trademark policies and more</span></a>
+            </li>
+            <li>
+                <a href="/contact"><span>Contact</span> <span>Get in touch with us</span></a>
+            </li>
+        </ul>
+    </li>
+    <li class="another_link">
+        <a href="https://developer.holochain.org/" target="_blank" rel="noopener noreferrer">DEV PORTAL</a>
+    </li>
+    <li class="another_link">
+        <a href="https://blog.holochain.org/" target="_blank" rel="noopener noreferrer">BLOG</a>
+    </li>
+
+    <li class="menu_item search_item">
+        <button class="gh-search-icon" aria-label="search" data-ghost-search style="display: inline-flex; justify-content: center; width: 20px; height: 20px; margin-top: 0; padding: 0; border: 0; color: inherit; background-color: transparent; cursor: pointer; outline: none;">
```

Comment:
I see that the identifiers that [your search JS targets](https://github.com/anisulkibria/holochain-blog-theme/blob/new-header/assets/js/custom.js#L14C20-L14C20) have been removed. Does search still work?

Reply:
conversation with Anisul elsewhere confirms that this is intentional.